diagnostic.md

```diff
@@ -57,13 +63,15 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+li.hide()
```

Comment:
Remember to use jQuery we need to start with the `$` sign or the `jQuery` keyword. So this should be:

`$('li').parent().parent().hide()`